docs/layers/mvt-layer.md

```diff
@@ -81,12 +81,25 @@ Inherits all properties from [`TileLayer`](/docs/layers/tile-layer.md) and [base
 If using the default `renderSubLayers`, supports all [`GeoJSONLayer`](/docs/layers/geojson-layer.md) properties to style features.
 
 
-##### `data` (String|Array)
+##### `data` (String | Array)
 
 Required. Either a URL template or an array of URL templates from which the MVT data should be loaded. See `TileLayer`'s `data` prop documentation for the templating syntax.
 
 The `getTileData` prop from the `TileLayer` class will not be called.
 
+##### `uniqueIdProperty` (String)
+
+Optional. Needed for highlighting a feature split across two or more tiles if no [feature id](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#42-features) is provided.
+
+An string pointing to a tile attribute containing a unique identifier for features across tiles.
+
+##### `highlightedFeatureId` (Number)
```

Comment:
Does the feature id have to be a number? Is string accepted? If so, the default should be `null` for clarity?

Reply:
As per MVT docs, feature ID can be [only a number](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/vector_tile.proto#L32). But it's true that if anyone uses `uniqueIdProperty` to reference an ID within feature properties, it could be a string.

I have just changed it to support strings as well in the code. Let me know if you see something weird.